go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/onsi/ginkgo v1.16.4
 	github.com/onsi/gomega v1.15.0
 	github.com/pkg/errors v0.9.1
+	github.com/sirupsen/logrus v1.8.1
```

Comment:
We didn't use logrus, we use `zap` let's not introduce a new logger here, that would be another feature entirely

Reply:
e4ab1b4aa88a80360a76182418078fd4accef7a3
removed